artifacts/ts/Assert.ts

```diff
@@ -24,14 +24,25 @@ import {
   ContractInstance,
   getContractEventsCurrentCount,
 } from "@alephium/web3";
-import { default as AssertContractJson } from "../test/assert.ral.json";
+import { default as AssertContractJson } from "../test/Assert.ral.json";
 
 // Custom types for the contract
 export namespace AssertTypes {
   export type State = Omit<ContractState<any>, "fields">;
 }
 
 class Factory extends ContractFactory<AssertInstance, {}> {
+  Error: bigint = BigInt(3);
+  A: bigint = BigInt(-3);
+  B: Address = "1DrDyTr9RpRsQnDnXo2YRiPzPW4ooHX5LLoqXrqfMrpQH";
+  C: HexString = "0011";
+  Addresses = {
+    A: "1DrDyTr9RpRsQnDnXo2YRiPzPW4ooHX5LLoqXrqfMrpQH",
+    B: "14UAjZ3qcmEVKdTo84Kwf4RprTQi86w2TefnnGFjov9xF",
+  };
+  Numbers = { A: BigInt(0), B: BigInt(1) };
+  ByteVecs = { A: "00", B: "11" };
```

Comment:
Would be more friendly if we wrap all these contants in an object, like:
```
consts = {
  Error: ...,
  A: ...,
  ...
}

we could use Assert.consts.{Error, A, B, ...} instead of Assert..{Error, A, B, ...}?
```